_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,284 @@
+                <option value="">- Choose one -</option>
+                <option value="Mary">Mary</option>
+                <option value="had">had</option>
+                <option value="a">a</option>
+                <option value="Little">Little</option>
+                <option value="Lamb">Lamb</option>
+            </select>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6">Radios</h4>
+<div class="example-set">
+    <fieldset class="fieldset form-group">
+        <legend class="group-legend form-label">Default radio buttons</legend>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="updates_mobile_text_01">
+                <input type="radio" class="input input-radio" name="updates_notifications_01" id="updates_mobile_text_01" value="Mobile text messages" aria-describedby="updates_mobile_text_01_helper" checked="checked">
+                <span class="input-label">Text messages on my mobile phone</span>
+            </label>
+            <div class="helper-text" id="updates_mobile_text_01_helper">Rates apply. Pelase consult your plan.</div>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="updates_email_01">
+                <input type="radio" class="input input-radio" name="updates_notifications_01" id="updates_email_01" value="Email messages" aria-describedby="updates_email_01_helper">
+                <span class="input-label">Email at matttracker@gmail.com</span>
+            </label>
+            <div class="helper-text" id="updates_email_01_helper">We will never spam you. Pinky swear.</div>
+        </div>
+    </fieldset>
+    <fieldset class="fieldset form-group">
+        <legend class="group-legend form-label">Enhanced radio buttons</legend>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="updates_mobile_text_02">
+                <input type="radio" class="input input-radio replace-radio" name="updates_notifications_02" id="updates_mobile_text_02" value="Mobile text messages" aria-describedby="updates_mobile_text_02_helper" checked="checked">
+                <span class="input-label">Text messages on my mobile phone</span>
+            </label>
+            <div class="helper-text" id="updates_mobile_text_02_helper">Rates apply. Pelase consult your plan.</div>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="updates_email_02">
+                <input type="radio" class="input input-radio replace-radio" name="updates_notifications_02" id="updates_email_02" value="Email messages" aria-describedby="updates_email_02_helper">
+                <span class="input-label">Email at matttracker@gmail.com</span>
+            </label>
+            <div class="helper-text" id="updates_email_02_helper">We will never spam you. Pinky swear.</div>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6">Checkboxes</h4>
```

Comment:
Add ``.example-set-hd`` class in here for consistency and extra PL Doc site formatting.